dev-libs/yaz/yaz-5.26.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+inherit eutils autotools
+
+DESCRIPTION="C toolkit for Z39.50v3 clients and servers"
+HOMEPAGE="http://www.indexdata.dk/yaz"
+SRC_URI="http://ftp.indexdata.dk/pub/${PN}/${P}.tar.gz"
+
+LICENSE="BSD GPL-2"
+SLOT="0"
+KEYWORDS="alpha amd64 arm hppa ~ia64 ~mips ~ppc ppc64 ~s390 ~sh ~sparc x86"
+IUSE="debug icu libressl tcpd ziffy"
+
+RDEPEND="dev-libs/libxml2
+        dev-libs/libxslt
+        !libressl? ( dev-libs/openssl:0= )
+        libressl? ( dev-libs/libressl:0= )
+        icu? ( dev-libs/icu:= )
+        tcpd? ( sys-apps/tcp-wrappers )
+        ziffy? ( net-libs/libpcap )"
+DEPEND="${RDEPEND}
+        virtual/pkgconfig
+        dev-lang/tcl:0
+        >=sys-devel/libtool-2"
+
+src_prepare() {
+        epatch "${FILESDIR}"/${PN}-5.26.1-icu-automagic.patch
+        AT_M4DIR="m4" eautoreconf
+}
+
+src_configure() {
+        econf \
+                --enable-static \
```

Comment:
how is that? you either enable static or shared, nobody wants static libs, so either bind it to the useflag or disable at all